.changeset/slow-pants-work.md

```diff
@@ -0,0 +1,7 @@
+---
+"@salt-ds/core": patch
```

Comment:
Let's split to 2 changeset file. One for core one for lab (when content is different for / irrelevant between each packages)